src/Symfony/Component/Testing/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "symfony/testing",
+    "type": "library",
+    "description": "Symfony Testing Component",
+    "keywords": [],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.3"
+    },
+    "autoload": {
+        "psr-0": { "Symfony\\Component\\Testing\\": "" },
+        "files": [ "bootstrap.php" ]
```

Comment:
this seems wrong to me. You want the bootstrap.php file to be loaded only when running the testsuite of your project, not when running your project locally (and you don't want to need to reinstall the project without dev requirements all the time during development